src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx

```diff
@@ -51,6 +53,14 @@ const MSG = defineMessages({
     id: 'dashboard.AddExistingSafeDialog.CheckSafe.safe',
     defaultMessage: 'Safe',
   },
+  callout: {
+    id: 'dashboard.AddExistingSafeDialog.CheckSafe.callout',
+    defaultMessage: '<span>Important!</span>  Read instuctions before starting',
```

Comment:
I blame myself, but there is a typo here in "instuctions", it should be "Important! Read instructions before starting"